packages/app-store/types.d.ts

```diff
@@ -8,6 +8,7 @@ export type IntegrationOAuthCallbackState = {
   returnTo: string;
   installGoogleVideo?: boolean;
   teamId?: number;
+  returnToOnboarding?: boolean;
```

Comment:
used to tell oAuth apps in callback endpoint to send back to the onboarding, redirec to onboarding should be done only if app extendsFeature "EventType"